packages/node/src/dev-server.ts

```diff
@@ -124,14 +119,14 @@ export async function onDevRequest(
   }
 
   try {
-    const result = await handleEvent(req);
-    res.statusCode = result.statusCode;
-    for (const [key, value] of Object.entries(result.headers)) {
-      if (typeof value !== 'undefined') {
+    const { headers, body, status } = await handleEvent(req);
+    res.statusCode = status;
+    for (const [key, value] of headers as Headers) {
```

Comment:
why is type cast necessary here? Can we update the return type of `handleEvent` to be more correct?

Reply:
It messing with global types because you are using the global type `Headers` in the `./types` file for `VercelProxyResponse`. There is a better solution here. I recommend looking for a better type to use in `VercelProxyResponse`.